src/main/kotlin/de/bund/digitalservice/useid/widget/WidgetController.kt

```diff
@@ -66,7 +66,7 @@ class WidgetController(
         )
 
         if (isIncompatibleOSVersion(userAgent)) {
-            return Rendering.redirectTo("/$INCOMPATIBLE_PAGE?hostname=$hostname").build()
```

Comment:
`hostname` is not used anymore, but we need to keep `@RequestParam() hostname: String` in the function param, if consumer of our endpoint is requesting to `/widget` without hostname, they will get `Bad Request` response rather than OK